core/includes/date.inc

```diff
@@ -608,7 +608,7 @@ function date_timezone_abbr($refresh = FALSE) {
  * Translates formatted date results, unlike PHP function date_format().
  * Should only be used for display, not input, because it can't be parsed.
  *
- * @param object $date
+ * @param BackdropDateTime|string|null $date
```

Comment:
I'm not sure, if this could be string|null.
Obviously, there's some check in the function to prevent problems when it's empty (which might happen when an error occurs when creating the date object, old code, though)... 🤔
But unlike format_date(), this wouldn't accept a timestamp. Or am I missing something?